src/types.jl

```diff
@@ -192,7 +202,16 @@ isexpr(s::BasicSymbolic) = !issym(s)
 iscall(s::BasicSymbolic) = isexpr(s)
 
 @inline isa_SymType(T::Val{S}, x) where {S} = x isa BasicSymbolic ? Unityper.isa_type_fun(Val(SymbolicUtils.BasicSymbolic), T, x) : false
+
+"""
+  issym(x)
+
+Returns `true` if `x` is a symbol. If true, `nameof` must be defined
+on `x` and must return a Symbol.
```

Comment:
```suggestion
    issym(x)

Returns `true` if `x` is a `Sym`. If true, `nameof` must be defined
on `x` and must return a `Symbol`.
```

- A four-space indent is needed so that it is printed as Julia code.
See https://docs.julialang.org/en/v1/manual/documentation/#Writing-Documentation
- "symbol" might be misunderstood with Julia [Symbols](https://docs.julialang.org/en/v1/manual/metaprogramming/#Symbols).